### ISSUE TITLE
IINA 1.0.7 Build 119 do not have real time thumbnail on progress bar and touch bar is totally gray when using smb / afp

### ISSUE THREAD

**Issue by darkaforest** [iina/iina#3108]
IINA 1.0.7 Build 119 do not have real time thumbnail on progress bar and touch bar is totally gray when using smb / afp
all function is ok if file is on the local SSD. is this a feature or a bug

IINA 1.0.7 Build 119 在使用smb / afp等文件协议时进度条上没有缩略图，touchbar灰色。播放本地文件没有此问题，我之前用的是很老很老的版本 是可以都正常显示的。

**Comment by p0we7** [comment]
我也是这样，之前以为系统问题，但是连硬件连系统都重装过几次了，依然没有实时缩略图。

我是用 Windows 的文件共享，macOS 连接到 smb 去播放的。

**Comment by lhc70000** [comment]
Please run this command in the terminal.

```
defaults write com.colliderli.iina enableThumbnailForRemoteFiles -bool YES
```

Currently this option is not available in the user interface.

**Comment by p0we7** [comment]
> Please run this command in the terminal.
> 
> ```
> defaults write com.colliderli.iina enableThumbnailForRemoteFiles -bool YES
> ```
> 
> Currently this option is not available in the user interface.

Yes, i just searched for this issue #2832 , and the thumbnails came up normally after setting it up.
